Resources/Prototypes/_NF/Guidebook/nf14.yml

```diff
@@ -10,3 +10,4 @@
   - Survival # Moved out from Station and Shifts section
   - Expeditions
   - Shipyard
+  - StartingGear
```

Comment:
Would this make more sense under "New? Start here!"
```
v New? Start here!
> Controls
> Creating Characters
> Frontier: Starting Gear
```
(newplayer.yml, under NewPlayer)

I think it'd make sense for the new player section to be your basic intro, and the frontier guide should be a bit more advanced or tied to Frontier-specific mechanics.

Reply:
Thanks! Was wondering how to get that page to show up as the one you see first.